app/konnect/centralized-consumer-management.md

```diff
@@ -43,11 +43,11 @@ You can manage consumers centrally using the {{site.konnect_short_name}} API. On
    Be sure to replace the following with your own values:
    * `{region}`: Region for your {{site.konnect_short_name}} instance. Data planes can only reach out to realms in the same region as the data plane.
    * `$KONNECT_TOKEN`: Replace with your {{site.konnect_short_name}} personal access token.
-   * `$CONTROL_PLANE_UUID`: (Optional) Replace with your control plane UUID.
+   * `$CONTROL_PLANE_UUID`: (Optional) Replace with your control plane UUID. If you don't specify a control plane, this means that no control plane can access the consumers in the realm.
    * `ttl`: (Optional) 'ttl' is the time-to-live (TTL) in minutes of the consumer for this realm in the {{site.base_gateway}} cache.
    * `negative_ttl`: (Optional) Represents the TTL of a bad login cache entry.
-   * `$CONSUMER_GROUP`: (Optional) Replace with the name of the consumer groups you want to associate with the realm.
-1. Use the `/realms/{realmId}/consumers` endpoint to create a centrally managed consumer and optionally assign it to a consumer group:
+   * `$CONSUMER_GROUP`: (Optional) Replace with the name of the consumer groups you want to associate with the realm. When you assign consumers to the realm, this will also associate them with the consumer groups you list here. Consumer groups set here are additive. This means that if you configure the realm with consumer groups A and B, and then configure the consumer with consumer group C, the authenticated consumer will be assigned to consumer groups A, B, and C.
```

Comment:
Consistent with "CONTROL_PLANE_ID" i would change the variable to `CONSUMER_GROUP_NAME` here and in the example command.